e2e/wrappers/tests/button.test.js

```diff
@@ -0,0 +1,8 @@
+import { Selector } from 'testcafe';
+import { testInFramework } from '../test-helpers';
+
+testInFramework('button', async (t) => {
```

Comment:
So, here `'button'` is the name of the page against we test, so that each test scenario for button has the same value here.

I guess tests should also have individual names to distinguish between them